File: PowerCraft/Tools/Updater.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace PowerCraft.Tools
{
    public class Updater
    {
        public static String VersionString()
        {
            return "PowerCraft " + versionNumber();
        }
        public static String versionNumber()
        {
            return "1.0.0.0";
        }
    }
}

File: PowerLib/PowerCraft/Tools/Updater.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace PowerCraft.Tools
    Public Class Updater
        Public Shared Function VersionString() As [String]
            Return "PowerCraft " + versionNumber()
        End Function
        Public Shared Function versionNumber() As [String]
            Return "1.0.0.0"
        End Function
    End Class
End Namespace